generation/WinSDK/enums.json

```diff
@@ -31932,5 +31957,48 @@
     },
     "members": [],
     "uses": []
+  },
+  {
+    "name": "EMPTY_VOLUME_CACHE_FLAGS",
+    "autoPopulate": {
+      "filter": "EVCF_",
+      "header": "emptyvc.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "inteface": "IEmptyVolumeCache2",
+        "method": "InitializeEx",
+        "parameter": "pdwFlags"
+      },
+      {
+        "inteface": "IEmptyVolumeCache",
+        "method": "Initialize",
+        "parameter": "pdwFlags"
+      },
+      {
+        "inteface": "IEmptyVolumeCache",
+        "method": "Deactivate",
+        "parameter": "pdwFlags"
+      }
+    ]
+  },
+  {
+    "name": "MW_FILTERMODE",
+    "autoPopulate": {
+      "filter": "MW_FILTERMODE_",
+      "header": "magnification.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "method": "MagSetWindowFilterList",
```

Comment:
I see Set in the diff but not Get.

Reply:
I will check it and rebuild again.

Reply:
The `MagGetWindowFilterList` `dwFilterMode` is a pointer. That could be why?

Reply:
I don't think that matters. I see the name of the parameter is `pdwFilterMode` in the winmd on my machine. Are you sure the you're using the right parameter name?

Reply:
That was the problem on this one. Rebuilding now.